grpc_opa/authorizer.go

```diff
@@ -353,6 +361,32 @@ func (o OPAResponse) Allow() bool {
 	return allow
 }
 
+// If reason is available, returns reason RBAC was forbidden
+func (o OPAResponse) rbacForbiddenReason() string {
```

Comment:
I think the signature needs work here. We should distinguish lack of obligations from a specific obligation denial. You could also do that by emitting an error instead of a string. `err == nil` nothing about obligations to report.

Also, how do services turn these logs off if they don't want them? Perhaps, don't log at all and return wrapped errors is a better way to enable implementation to choose behavior.